Userland/Applications/FileManager/DirectoryView.cpp

```diff
@@ -580,8 +580,8 @@ void DirectoryView::handle_drop(const GUI::ModelIndex& index, const GUI::DropEve
         if (url_to_copy.path() == new_path)
             continue;
 
-        if (!FileUtils::copy_file_or_directory(url_to_copy.path(), new_path)) {
-            auto error_message = String::formatted("Could not copy {} into {}.", url_to_copy.to_string(), new_path);
+        if (auto rc = Core::File::copy_file_or_directory(new_path, url_to_copy.path()); rc.is_error()) {
+            auto error_message = String::formatted("Could not copy {} into {}: \n", url_to_copy.to_string(), new_path, rc.error());
```

Comment:
`rc` is conventionally used to store syscall return codes.
Since we're not calling a syscall here, I'd call it `result` instead.

Also the format string looks wrong, missing a "{}"
And we don't need to end it with `\n` if we're showing it in a **GUI::MessageBox** :)